src/Datasource/RepositoryInterface.php

```diff
@@ -74,7 +74,7 @@ public function find($type = 'all', $options = []);
      * @return \Cake\Datasource\EntityInterface
      * @see \Cake\Datasource\RepositoryInterface::find()
      */
-    public function get($primaryKey, $options = []);
+    public function get($primaryKey, $options = []): EntityInterface;
```

Comment:
If getPrimaryKey() is array every $primaryKey type hint should be, too?

Reply:
It can be array or string afaik. That's the dirtyness here in the return value of this method.